pyuvdata/uvdata/uvdata.py

```diff
@@ -12565,7 +12566,7 @@ def read(
                 "The `make_multi_phase` option is deprecated. Since all objects now "
                 "support multiple phase centers, the only effect of setting this to "
                 "True is to prevent rephasing. A better approach is to set "
-                "`allow_rephase=False`.",
+                "`allow_rephase=False`. This will become an error in version 3.0.",
```

Comment:
Maybe worth adding to milestone list if not already.

Reply:
It's there already.